cnd/src/db/save.rs

```diff
@@ -371,21 +371,12 @@ struct InsertableEthereumBitcoinAcceptMessage {
     bitcoin_refund_identity: Text<bitcoin::PublicKey>,
 }
 
-#[impl_template]
 #[async_trait]
-impl
-    Save<
-        Accept<
-            Ethereum,
-            ((
-                ledger::bitcoin::Mainnet,
-                ledger::bitcoin::Testnet,
-                ledger::bitcoin::Regtest,
-            )),
-        >,
-    > for Sqlite
-{
-    async fn save(&self, message: Accept<Ethereum, __TYPE0__>) -> anyhow::Result<()> {
+impl Save<Accept<crate::ethereum::Address, crate::bitcoin::PublicKey>> for Sqlite {
```

Comment:
OMG SO NICE!

Reply:
This is exactly the kind of benefits I expected from these changes.
Basically impossible to see upfront but things turn out super nicely :)